bgpd/bgpd.c

```diff
@@ -6021,180 +6021,184 @@ int peer_weight_unset(struct peer *peer, afi_t afi, safi_t safi)
 	return 0;
 }
 
-int peer_timers_set(struct peer *peer, uint32_t keepalive, uint32_t holdtime)
+int peer_timers_set(struct peer *peer, uint32_t val1, uint32_t val2,
```

Comment:
would this support dampening, etc.? are there cases where we would need three values, and how much of the code would need to be touched if so?